packages/engine/bin/cli/Cargo.toml

```diff
@@ -20,6 +20,4 @@ tokio = "1.19.2"
 uuid = { version = "1.1.2", features = ["v4", "serde"] }
 
 [features]
-default = ["build-nng"]
```

Comment:
This PR feels like we're doing more than "Updating Python Dependencies" and I'm a little concerned it might be difficult to find in the future. Can we update the PR title, and be careful when clicking merge to update the commit contents too?